README.md

```diff
@@ -441,6 +441,21 @@ The Strategy includes a few options that can be customized.
 
 You can find a detailed list of all the available options in the [customization](https://github.com/dev-xo/remix-auth-totp/blob/main/docs/customization.md) documentation.
 
+## Cloudflare
```

Comment:
Could we also move this to `customization.md` or its own file? I would like to keep the initial readme as simple as possible, as it's already quite extensive.

Also, should the `remix.config` be a JS file instead of a TS? Is that the case for Cloudflare? If not, it would be great to have a TS refactor. 🙏

Reply:
`remix.config` seems to be a JS file as far as I can tell per documentation (https://remix.run/docs/en/main/file-conventions/remix-config) and also what is generated with remix cloudflare pages template. Moved the Cloudflare section from the `README.md` to `customization.md` and added little blurb.